websites/B/backpack.tf/dist/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "Backpack.tf",
+  "author": {
+    "name": "a＠a",
+    "id": "405028899798384640"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/YtdujIW.png",
+  "thumbnail": "https://i.imgur.com/uQCICM4.png",
+  "version": "1.0.1",
```

Why are you starting on 1.0.1?
```suggestion
  "version": "1.0.0",
```